client/src/pages/ClientSignupPage.js

```diff
@@ -1,21 +1,272 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import "../App.css";
 import React from "react";
 import { Container, Row } from "react-bootstrap";
-import { employeeNavbarLinks } from "../Routes";
 import { NavbarComponent } from "../ui-components/NavbarComponent/NavbarComponent";
-
-import "bootstrap/dist/css/bootstrap.min.css";
+import dayjs from "dayjs";
+import { useState, useEffect } from "react";
+import { Button, Col, Form ,Alert, } from "react-bootstrap";
+import { Link, Redirect, Route, Switch, useLocation } from "react-router-dom";
+//import {loadAllClients, addClient} from "../services/ApiClient"
 
 function ClientSignupPage(props) {
```

Comment:
Are you sure should be called ClientSignupPage? In the next sprint probably we will need to implement the client signup done by the client itself and not by the employee